build.rs

```diff
@@ -1,14 +1,11 @@
 #[cfg(target_arch = "x86_64")]
-fn main() {
```

Comment:
please revert, this will trigger issues when building for things like `wasm`

Reply:
Just merged the file from `master` branch. Done with 02b5f8d.